core/src/test/java/org/hibernate/ogm/backendtck/queries/SimpleQueriesTest.java

```diff
@@ -313,6 +313,37 @@ public void testNotInQueryReturnsEntityWithQueriedPropertySetToNull() throws Exc
 	}
 
 	@Test
+	@TestForIssue(jiraKey = "OGM-581")
+	public void testParameterList() throws Exception {
+		List<String> paramList = Arrays.asList( "Lama", "Puma" );
+		List<?> result = session.createQuery( "from Helicopter h where h.name IN (:names)" ).setParameterList( "names", paramList ).list();
+		assertThat( result ).onProperty( "name" ).containsOnly( "Lama" );
+	}
+
+	@Test
+	@TestForIssue(jiraKey = "OGM-581")
+	public void testParameterListWithLong() throws Exception {
+		//List<Long> paramList = Arrays.asList( 2L );
```

Comment:
You are right. It's a left-over. Sorry.

I will improve the tests to request several values, some of which will be present and others won't.